### PR TITLE
Fixes and cleanup found while reviewing code

### PR DESCRIPTION
**Pull request by mpenick**

### CODE REVIEW

File: src/utils.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@
 #include <functional>
 #include <sstream>
 
-#if defined(_MSC_VER)
+#if (defined(WIN32) || defined(_WIN32))
```
Comment (stamhankar999) [inline src/utils.cpp]: Is WIN32 defined for 64-bit Windows? We need windows.h for both.

**Comment by stamhankar999** [comment]
Other than my question about 64-bit Windows, lgtm.

**Comment by mpenick** [comment]
`WIN32` means the Windows API. It's always defined regardless of the architecture: https://msdn.microsoft.com/en-us/library/b0084kay.aspx

**Comment by stamhankar999** [comment]
👍  Merge at will.

**Comment by mpenick** [comment]
Merged at 8e7b256023d40830730e1cb24978f5a9b658d064

**Pull request closed without merging**